src/components/StepButton/StepButton.tsx

```diff
@@ -0,0 +1,54 @@
+import type { FC, MouseEvent } from 'react';
+import cn from 'classnames';
+
+type Props = {
+  direction: 'next' | 'prev';
```

Comment:
better to create enum if we exactly know the string values it's gonna have. That decreases a chance of missing to impossible. Making it a string though, looks kinda hardcoded